Automattic/wp-calypso
# Complete installation of marketplace dotorg paid plugins

### Proposed Changes

Fixed thank you page for marketplace dotorg plugins like the paid Visual Composer

### Testing Instructions




- Make sure you don't have Visual Composer installed yet
- Install the [Visual Composer wpcom](http://calypso.localhost:3000/plugins/visualcomposer-wpcom) plugin at your site
- Make sure the thank you page confirms the plugin installation:

	![image](https://user-images.githubusercontent.com/1044309/194077391-80c7418b-7c38-4f5e-9d37-58add318278c.png)

- Make sure wccom marketplace plugins like `WooCommerce Subscriptions` also confirm installation as expected
- Make sure dotorg plugins like `Booking Calendar` also confirm installation as expected



Related to https://github.com/Automattic/wp-calypso/issues/68610
